type-tests/preview/@ember/owner-tests.ts

```diff
@@ -35,19 +41,19 @@ aFactory.create({
   hasProps: false,
 });
 
-// These should be rejected by way of EPC
-// @ts-expect-error
+// NOTE: it would be nice if these could be rejected by way of EPC, but alas: it
+// cannot, because the public contract for `create` allows implementors to
+// define their `create` config object basically however they like. :-/
 aFactory.create({ unrelatedNonsense: 'yep yep yep' });
-// @ts-expect-error
 aFactory.create({ hasProps: true, unrelatedNonsense: 'yep yep yep' });
 
 // But this should be legal.
 const goodPojo = { hasProps: true, unrelatedNonsense: 'also true' };
 aFactory.create(goodPojo);
 
-// while this should be rejected for *type error* reasons, not EPC
+// This should also be rejected, though for *type error* reasons, not EPC; alas,
+// it cannot, for the same reason.
```

Comment:
~Wasn't there a change in an earlier commit from something like `create(props: object)` to `create(props: Partial<T>)`? I would expect that to reject `hasProps: 'huzzah'` here.~

Nevermind, I see that got undone in 7cc146db09c5969349eefe6221c1a5ae603be644